ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.java

```diff
@@ -69,6 +69,21 @@ public void setTextAlign(ReactTextView view, @Nullable String textAlign) {
       throw new JSApplicationIllegalArgumentException("Invalid textAlign: " + textAlign);
     }
   }
+  
+  @ReactProp(name = "textDecorationLine")
+    public void setTextDecorationLine(ReactTextView view, @Nullable String textDecorationLine) {
+        if (textDecorationLine == null || "none".equals(textDecorationLine)) {
+            view.getPaint().setFlags(0);
+        } else if ("underline".equals(textDecorationLine)) {
+            view.getPaint().setFlags(Paint.UNDERLINE_TEXT_FLAG | Paint.ANTI_ALIAS_FLAG);
+        } else if ("line-through".equals(textDecorationLine)) {
+            view.getPaint().setFlags(Paint.STRIKE_THRU_TEXT_FLAG | Paint.ANTI_ALIAS_FLAG);
+        } else if ("underline line-through".equals(textDecorationLine)) {
```

Comment:
this also doesn't handle a reordering of underline/line-through... I would think that we should actually be sending an array of strings or a bit-wise anded int to send what props are set from JS